content/How an engineer can be your best partner.md

```diff
@@ -0,0 +1,86 @@
+
+`Data analysis is defined as a process of cleaning, transforming, and modeling data to discover useful information for business decision-making. The purpose of Data Analysis is to extract useful information from data and taking the decision based upon the data analysis`
+
+You may see that `data` word came up a lot, that's not because I want to be redundant, instead is because I want to emphasize it as a really important concept. Nowadays the more data you have the more power you have to make better decisions.
+
+
+## 2) How to start?
+
+As I mentioned before, start gathering data, it doesn't need to be super high-quality data, you will start learning from your users and your business reality and improve step by step.
+A simple step you can take at a really low cost is adding some forms and analytics to your landing page. This way you will start getting feedback from users and also start knowing your users interactions and interests.
+
+First, see an initial approach on how you should approach your changes:
+![hypothesis process](images/build-measure-learn.jpg)
+
+You will start building something as small as possible (to lessen the number of mistakes you can make).
+Then you measure it (please! test it with users, they are the only ones with the truth on this).
+Finally, learn from your measures and keep building, but now with a clearer idea of what users want.
+
+To plan what to build you can follow this simple process, also to manage each specific change.
+![hypothesis process](images/hypothesis.jpg)
+
+  - Write down your hypothesis based on your business goals
+  - Plan tests on your product to validate your hypothesis
+  - Measure
+  - Verify your numbers against what you thought
+
+The most important step at the end is to make decisions based on your learnings, adjusting your upcoming features/hypothesis based on learnings.
+
+## 3) Common mistakes
+
+#### First and most usual mistake: assume you know how users will behave
+
+You will probably be wrong, instead of that write a hypothesis and try to prove it. It can be as simple as "I want to prove that I get more sales if I place the BUY button in some place with some color" or more complex like "I think users are tempted to buy if I offer discounts after 10 minutes looking items in my store"
+
+#### Misread data
+
+Actually is really easy to fall on this error, as you have an idea in your mind and naturally, you want it to end up with that being truth.
+So my proposal for this is to again write down your ideas based on hypothesis and set up expected metrics for each of them, so you can properly check against something objectively.
+
+#### Think you have finished
+
+This is a continuous delivery process that never ends. User preferences changes and you can always detect new opportunities or tends to look at data.
+I see this as a simple process that prevents you to take wrong decisions, so you only can win from using it.
```

Comment:
Maybe talk about how if you are sure about a feature you can do a `90/10` or something similar just to be sure or the other way around, testing risky features with only a portion of your users